wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'reseed',
+  'seed' => 'create',
+  'load' => 'seed'
+}
```

Comment:
yes, we can change the task names so long as we support the same core CLI commands (wagn new, wagn update, etc).  you can even use decko for the task namespace afaic.